posts/unitary_labs_intro.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Unitary Fund</title>
+    <!-- CSS -->
+    <link rel="stylesheet" type="text/css" href="../style.css">
+
+    <link href="//cdn-images.mailchimp.com/embedcode/horizontal-slim-10_7.css" rel="stylesheet" type="text/css">
+    <style type="text/css">
+        #mc_embed_signup{background:#fff; clear:left; font:14px "Lucida Console", Monaco, monospace; width:100%;}
+        #mce-EMAIL{padding: 11px; margin: 10px;}
+
+    </style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-19932157-4"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-19932157-4');
+
+    </script>
+</head>
+
+<body>
+<div class="layout">
+    <main id="content" class="content">
+        <section class="heavy">
+
+            <center>
+                <h1><a style="text-decoration:none" href="../index.html">Unitary Fund</a></h1>
+
+                <p class="tagline"><b>Because evolution is
+                    <a href="https://en.wikipedia.org/wiki/Unitarity_(physics)" target="_blank">unitary</a></b>.</p>
+
+            </center>
+
+            <p class="light">
+                <a href="../index.html#supporters">supporters</a> |
+                <a href="../index.html#team">team</a> |
+                <a href="../index.html#grants-made">grants made</a> |
+                <a href="../index.html#project-ideas">project ideas</a> |
+                <a href="../index.html#faq">faq</a> |
+                <a href="../blog.html">blog</a>
+            </p>
+
+            <p><b>Unitary Fund</b> is a non-profit working to create a quantum technology ecosystem that benefits the
+                most people.
+            </p>
+
+
+        </section>
+        <section class="light">
+            <br>
+            <br>
+
+            <center>
+                <h1>Introducing the Technical Team: Unitary Labs</h1>
+            </center>
+            <br>
+            <br>
+            14 March, 2020<br>
```

Comment:
Maybe we could change this to 19 March, 2020 (or actual release date). Also in the main blog directory.

Reply:
removed